html/manifest/note_taking.json

```
@@ -0,0 +1,104 @@
+{
+  "html": {
+    "manifest": {
+      "note_taking": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/Manifest/note_taking",
+          "spec_url": "https://wicg.github.io/manifest-incubations/#note_taking-member",
+          "support": {
+            "chrome": {
+              "version_added": "95"
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": "95"
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "new_note_url": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/Manifest/new_note_url",
```

I guess this should either be a sub page

```suggestion
            "mdn_url": "https://developer.mozilla.org/docs/Web/Manifest/note_taking/new_note_url",
```

Or a section on the note_taking page.

```suggestion
            "mdn_url": "https://developer.mozilla.org/docs/Web/Manifest/note_taking#new_note_url",
```